modules/setting/setting.go

```diff
@@ -197,12 +197,13 @@ var (
 	ScriptType   = "bash"
 
 	// UI settings
-	UI = struct {
-		ExplorePagingNum   int
-		IssuePagingNum     int
-		FeedMaxCommitNum   int
-		ThemeColorMetaTag  string
-		MaxDisplayFileSize int64
+	UI struct {
+		ExplorePagingNum       int
+		IssuePagingNum         int
+		FeedMaxCommitNum       int
+		ThemeColorMetaTag      string
+		MaxDisplayFileSize     int64
+		ShowUserEmailInExplore bool
```

Comment:
Should probably be `ShowUserEmail`, and changed accordingly in the rest of the patch :) (since this will be used for more than just Explore in the long run :)